extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -105,6 +105,7 @@ function _getDisplaySetsFromSeries(
     StudyInstanceUID,
     SOPClassHandlerId,
     SOPClassUID,
+    others: instances,
```

Comment:
is `others` a well-known dicom thing? it is too general. `instances` instead?

Reply:
others is what is used consistently in the getSopClassHandler modules for things other than images.  I agree that instances would be nicer, but would need to rename a bunch of them.